Vostok.Logging.Microsoft/VostokLoggerExtensions.cs

```diff
@@ -29,5 +29,15 @@ public static ILoggingBuilder AddVostok([NotNull] this ILoggingBuilder builder,
             builder.AddProvider(new VostokLoggerProvider(log));
             return builder;
         }
+
+        /// <summary>
+        /// <para>Create a new <see cref="MicrosoftLog"/> with given <paramref name="categoryName"/> from the <paramref name="factory"/></para>
+        /// </summary>
+        [NotNull]
+        public static MicrosoftLog CreateLog([NotNull] this ILoggerFactory factory, string categoryName)
```

Comment:
Because we return concrete implementation, I think we should rename method to something like **CreateVostokLog** or **CreateVostokMicrosoftLog**. This will help not to confuse it with built in **CreateLogger** method.

Reply:
I think it is more confusing to use **CreateMicrosoftLog** because ILoggerFactory is already about creating microsoft loggers via **CreateLogger** metrod (type ILogger belongs to Microsoft).

Moreover, it is a Vostok library, and It's better to reflect that we create exactly a Vostok logger, and word "Vostok" should be present in method's name. Maybe is is better to return not MicrosoftLog but ILog? So **CreateVostokLog**  name will be no more confusing for you.

Reply:
After writing this comment I found your idea **CreateVostokMicrosoftLog** satisfying all requirements. Commit is done.